### DISCUSSION TITLE
Can I install Whisky on an external disk?

### DISCUSSION THREAD

**Discussion by Lucas04-nhr** [Whisky-App/Whisky#477] [Q&A] [answered]
I want to install the Whisky App and the bottles to my external SSD, but I don't quite sure whether the app supports it or not.

**Comment by mavethee** [comment, marked as answer]
You can do symlinks:

**_For example:_**
Install Whisky, then run it, 
Create folder Whisky on external drive,

**_Create symlink:_** 
```sh
ln -sf /path/to/external/drive/Whisky ~/Library/Containers/com.isaacmarovitz.Whisky
```

**_If you're unsure about exact path on external drive or you're new to Terminal:_**
Open Finder and go to Whisky folder in desired location, 
Press `Option` + `Command` + `P` and on the bottom you will see path where you're located,
Right-click, `Copy "Whisky" as pathname` and replace with `/path/to/external/drive/Whisky` in my example.

This will store Whisky's content on external drive.
App can be left in `/Applications` as it's small.

That's the temporal solution, my advice is to opening an issue with Feature Request:
https://github.com/Whisky-App/Whisky/issues/new/choose
